pages/en/cookbook/base.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Building Subgraphs on Base
+---
+
+This guide will quickly take you through how to initialize, create, and deploy your subgraph on Base testnet.
+
+What you'll need:
+
+- A Base testnet contract address
+- A crypto wallet (e.g. MetaMask or Coinbase Wallet)
+
+## Subgraph Studio
+
+### 1. Install the Graph CLI
+
+The Graph CLI (>=v0.41.0) is written in JavaScript and you will need to have either `npm` or `yarn` installed to use it.
+
+```sh
+# NPM
+npm install -g @graphprotocol/graph-cli
+
+# Yarn
+yarn global add @graphprotocol/graph-cli
+```
+
+### 2. Create your subgraph in the Subgraph Studio
+
+Go to the [Subgraph Studio](https://thegraph.com/studio/) and connect your crypto wallet.
+
+Once connected, click "Create a Subgraph" and enter a name for your subgraph.
+
+Select "Base (testnet)" as the network and click Create Subgraph.
```

Comment:
In studio we use copy "INDEXED BLOCKCHAIN" instaed of "network". Just to be precise I'd say
"Select "Base (testnet)" as the indexed blockchain ...."

Reply:
Agreed, thank you!